projects/plugins/jetpack/extensions/blocks/recipe/recipe.php

```diff
@@ -9,55 +9,67 @@
 
 use Automattic\Jetpack\Blocks;
 
-Blocks::jetpack_register_block(
-	__DIR__,
-	array(
-		'render_callback' => array( 'Automattic\\Jetpack\\Extensions\\Recipe\\Jetpack_Recipe_Block', 'render' ),
-	)
-);
+add_action(
+	'init',
+	function () {
+		Blocks::jetpack_register_block(
+			__DIR__,
+			array(
+				'render_callback' => array( 'Automattic\\Jetpack\\Extensions\\Recipe\\Jetpack_Recipe_Block', 'render' ),
+			)
+		);
 
-Blocks::jetpack_register_block(
-	'jetpack/recipe-details',
-	array(
-		'parent' => array( 'jetpack/recipe' ),
-	)
-);
+		Blocks::jetpack_register_block(
+			__DIR__,
+			array(
+				'render_callback' => array( 'Automattic\\Jetpack\\Extensions\\Recipe\\Jetpack_Recipe_Block', 'render' ),
+			)
+		);
```

Comment:
Is it intended that you're making the same call twice here?

Reply:
No! Thank you.